src/style.css

```diff
@@ -1 +1,42 @@
-/* add styles here */
+body {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+  list-style-type: none;
+  margin-block-start: 0;
+  padding-inline-start: 0;
+  margin: 0;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-image: url(images/star.svg);
```

Comment:
Ensure that the `images/star.svg` file is present in the `images` folder. This image is essential for the star rating system to display correctly.